### DISCUSSION TITLE
Options is over Volume Control

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#16500] [Q&A] [answered]
I know you can adjust the volume from the options itself but I personally find this wrong.

https://user-images.githubusercontent.com/64581009/149919230-664f40ef-9914-49e1-ac46-3548991acac1.mp4

I think it should be above options since it's a quick-overlay afterall

**Comment by peppy** [comment]
The current ordering is how I intended it to be.

**Comment by frenzibyte** [comment, marked as answer]
On top of that, there is already an issue thread open specifically for this (https://github.com/ppy/osu/issues/11077), and a proposal to update the volume overlay design to avoid this issue (https://github.com/ppy/osu/issues/3413).
